### ISSUE TITLE
Add a color picker option for gradient

### ISSUE THREAD

**Issue by soustab10** [narayan954/dummygram#72]
### Description

Any post generated by the user only has this background:

Adding a color picker tool will be really beneficial to make the posts more unique.

### Screenshots

![image](https://user-images.githubusercontent.com/95632583/194175081-0599ac68-cde7-4c70-af39-8ced87babe95.png)

**Comment by narayan954** [comment]
Yeah, that'd be great. Would you like to implement that? @soustab10

**Comment by soustab10** [comment]
> Yeah, that'd be great. Would you like to try that? @soustab10

yes sure. Please assign it to me.

**Comment by soustab10** [comment]
In case of a gradient post, how is the background image generated? Is it API?

**Comment by narayan954** [comment]
The background gradient is hardcoded in the CSS file, (post__background class). So, in either case of post(with and without image), the background is gradient. @soustab10

**Comment by soustab10** [comment]
I need an API/Tool which will create a background image having the gradient with colors as chosen by the user, and then using that image as the background image of the post. Do you have anything in mind? Or any general suggestions. @narayan954

**Comment by narayan954** [comment]
@soustab10 do you mean something like this? [react gradient picker](https://www.npmjs.com/package/react-color-gradient-picker)

**Comment by soustab10** [comment]
Okay will try to implement.

**Comment by narayan954** [comment]
Open for KWOC

**Comment by devika-singh** [comment]
@narayan954 Pls assign me this issue

**Comment by narayan954** [comment]
> @narayan954 Pls assign me this issue

go ahead :)

**Comment by AyushK-101** [comment]
@narayan954 Can I work on this?

**Comment by narayan954** [comment]
> @narayan954 Can I work on this?

sure go ahead :)

**Comment by narayan954** [comment]
Hi @AyushK-101 , are you still working?

**Comment by priyanka1551** [comment]
I want to work on this issue under JWOC

**Comment by narayan954** [comment]
sure @priyanka1551 you can go ahead!

**Comment by priyanka1551** [comment]
@narayan954 tell me one thing ..do I need to add color picker for background or for user post only?

**Comment by narayan954** [comment]
> @narayan954 tell me one thing ..do I need to add color picker for background or for user post only?

You can add it for the posts

**Comment by priyanka1551** [comment]
> > @narayan954 tell me one thing ..do I need to add color picker for background or for user post only?
> 
> You can add it for the posts

Okay I am working on it

**Comment by Palakkgoyal** [comment]
Hey @narayan954 Can I work on this?

**Comment by narayan954** [comment]
> Hey @narayan954 Can I work on this?

sure, go ahead!
